ultranest/utils.py

```diff
@@ -441,3 +441,18 @@ def verify_gradient(ndim, transform, loglike, gradient, verbose=False, combinati
             print("expectation was L=", Lexpected, ", given", Lref, grad, eps)
         assert np.allclose(Lprime, Lexpected, atol=0.1 / ndim), \
             (u, uprime, theta, thetaprime, grad, eps * grad / L, L, Lprime, Lexpected)
+
+def todo_points_for_this_process(mpi_rank, num_live_points_missing, mpi_size):
```

Comment:
Please make `num_live_points_missing` first argument, and call it `num_total`

Reply:
Please call the function something more neutral (`points` is a bit out of context here) 

Perhaps `get_distributed_work_chunk_size`?

Reply:
`distributed_work_chunk_size` -- `get` seems redundant to me